sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -2136,139 +2116,11 @@ def get_transceiver_status(self):
-        vcchighalarm_flag            = BOOLEAN                          ; vcc high alarm flag
-        vcchighwarning_flag          = BOOLEAN                          ; vcc high warning flag
-        vcclowalarm_flag             = BOOLEAN                          ; vcc low alarm flag
-        vcclowwarning_flag           = BOOLEAN                          ; vcc low warning flag
-        txpowerhighalarm_flag        = BOOLEAN                          ; tx power high alarm flag
-        txpowerlowalarm_flag         = BOOLEAN                          ; tx power low alarm flag
-        txpowerhighwarning_flag      = BOOLEAN                          ; tx power high warning flag
-        txpowerlowwarning_flag       = BOOLEAN                          ; tx power low alarm flag
-        rxpowerhighalarm_flag        = BOOLEAN                          ; rx power high alarm flag
-        rxpowerlowalarm_flag         = BOOLEAN                          ; rx power low alarm flag
-        rxpowerhighwarning_flag      = BOOLEAN                          ; rx power high warning flag
-        rxpowerlowwarning_flag       = BOOLEAN                          ; rx power low warning flag
-        txbiashighalarm_flag         = BOOLEAN                          ; tx bias high alarm flag
-        txbiaslowalarm_flag          = BOOLEAN                          ; tx bias low alarm flag
-        txbiashighwarning_flag       = BOOLEAN                          ; tx bias high warning flag
-        txbiaslowwarning_flag        = BOOLEAN                          ; tx bias low warning flag
-        lasertemphighalarm_flag      = BOOLEAN                          ; laser temperature high alarm flag
-        lasertemplowalarm_flag       = BOOLEAN                          ; laser temperature low alarm flag
-        lasertemphighwarning_flag    = BOOLEAN                          ; laser temperature high warning flag
-        lasertemplowwarning_flag     = BOOLEAN                          ; laser temperature low warning flag
-        prefecberhighalarm_flag      = BOOLEAN                          ; prefec ber high alarm flag
-        prefecberlowalarm_flag       = BOOLEAN                          ; prefec ber low alarm flag
-        prefecberhighwarning_flag    = BOOLEAN                          ; prefec ber high warning flag
-        prefecberlowwarning_flag     = BOOLEAN                          ; prefec ber low warning flag
-        postfecberhighalarm_flag     = BOOLEAN                          ; postfec ber high alarm flag
-        postfecberlowalarm_flag      = BOOLEAN                          ; postfec ber low alarm flag
-        postfecberhighwarning_flag   = BOOLEAN                          ; postfec ber high warning flag
-        postfecberlowwarning_flag    = BOOLEAN                          ; postfec ber low warning flag
-        ================================================================================
+            Dictionary
```

Comment:
@mihirpat1 Add comment in code what this status means? as we have status flag also

Reply:
@prgeor Addressed this now